exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,16 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function shoutyName(name) {
+  return name.toUpperCase();
+}
+
+let mentors = [mentor1, mentor2, mentor3, mentor4, mentor5];
+
+function shoutyGreeting() {
+  for (i = 0; i < 5; i++) {
```

Comment:
The for loop works fine for now but what if the mentors array gets updated and the length is changed from 5? Let's say you or another developer in your team push/unshift more mentors into the array.  For this and similar reasons, your function won't break if you use `i < mentors.length` instead of a fixed number.